includes/Modules/Tag_Manager.php

```diff
@@ -93,6 +93,8 @@ public function register() {
 
 		// Tag Manager tag placement logic.
 		add_action( 'template_redirect', $this->get_method_proxy( 'register_tag' ) );
+		// Filter the Analytics useSnippet value.
+		add_action( 'googlesitekit_analytics_can_use_snippet', $this->get_method_proxy( 'can_analytics_use_snippet' ) );
```

Comment:
It would be great if we could add a quick test for this in `Tag_ManagerTest` to cover all the combinations of `gaPropertyID` and `$settings['useSnippet']`.

Reply:
@aaemnnosttv if you're able to add this that would be great as my time remaining is limited.

Reply:
Will do! 👍

Reply:
It ended up being a little less quick than I thought but should be good now 😄